skf/markdown/code_examples/java/11-code_example--enforce_secure_passwords--.md

```diff
@@ -0,0 +1,103 @@
+
+Enforce Secure Password
+------------------------
+
+***Example:***
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+package com.edw;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.log4j.Logger;
+
+public class PasswordCheck {
+	
+	final static Logger logger = Logger.getLogger(sessionhijack.class);
```

Comment:
The logger class should reference to the same class where it's used the log, in this case should be PasswordCheck.class